src/Pages/FooterTerms/CancellationPolicy/index.tsx

```diff
@@ -0,0 +1,8 @@
+import { FooterTerms } from "../PageLayout/index";
```

Comment:
`PageLayout` is a component so it should be within `../components/PageLayout`

Reply:
done